osmtm/templates/home.mako

```diff
@@ -170,7 +170,7 @@ sorts = [('priority', 'asc', _('High priority first')),
           title="${_('Access to this project is limited')}"></span> -
     % endif
     % if project.author:
-    <span>${_('Created by')} ${project.author.username}</span> -
+    <span>${_('Created by')} <a href="/user/${project.author.username}">${project.author.username}</a></span> -
```

Comment:
It's important to use the pyramid routing here otherwise instances with urls like `http://domain.com/myinstance/` will not work.